frontend/src/metabase/components/FieldValuesWidget/FieldValuesWidget.tsx

```diff
@@ -483,7 +484,7 @@ export function FieldValuesWidgetInner({
     !disableList &&
     shouldList({ parameter, fields, disableSearch }) &&
     valuesMode === "list";
-  const isLoading = loadingState === "LOADING";
+  const isLoading = loadingState !== "LOADED";
```

Comment:
If you look at stats right now, you'll see that if you watch carefully and click a parameter popover, you'll see that it loads the `MultiAutocomplete` component _really_ quickly, then the loading spinner, and then the correct component. Changing this to `LOADED` ensures that `INIT` is also accounted for in the loading state. This prevents the huge ternary statement in the render function from loading anything besides `LoadingState` until everything is ready